app/__init__.py

```diff
@@ -15,9 +15,9 @@
 from .upload.views import upload_blueprint
 
 
-def create_app(environment):
+def create_app(environment=None):
```

Comment:
can we remove this parameter entirely, and then add `FLASK_ENV=test` to the pytest.ini?

Reply:
Yeah, I wasn't sure about this fallback either. pytest.ini makes sense.